EIPS/eip-3675.md

```diff
@@ -0,0 +1,244 @@
+---
+eip: 3675
+title: Upgrade consensus to Proof-of-Stake
+author: Mikhail Kalinin (@mkalinin), Danny Ryan (@djrtwo), Vitalik Buterin (@vbuterin)
+discussions-to: https://ethereum-magicians.org/t/eip-3675-upgrade-consensus-to-proof-of-stake/6706
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-07-22
+requires: 2982
+---
+
+
+## Simple Summary
+
+Specification of the consensus mechanism upgrade on Ethereum Mainnet that introduces Proof-of-Stake.
+
+
+## Abstract
+
+This EIP deprecates Proof-of-Work (PoW) and supersedes it with the new Proof-of-Stake consensus mechanism (PoS) driven by the beacon chain. Information on the bootstrapping of the new consensus mechanism is documented in [EIP-2982](/EIPS/eip-2982). Full specification of the beacon chain can be found in [eth2.0-specs](https://github.com/ethereum/eth2.0-specs/) repository.
+
+This document specifies the set of changes to the block structure, block processing, the fork choice rule and the network interface introduced by the consensus upgrade.
+
+
+## Motivation
+
+The beacon chain network has been up and running since December 2020. Neither safety nor liveness failures were detected during this period of time. The long period of running without failures demonstrates sustainability of the beacon chain system and witnesses its readiness to start driving and become a security provider for the Ethereum Mainnet.
+
+To understand the motivation of introducing the Proof-of-Stake consensus see Motivation section of [EIP-2982](/EIPS/eip-2982#motivation).
```

Comment:
```suggestion
To understand the motivation of introducing the Proof-of-Stake consensus see Motivation section of [EIP-2982](/EIPS/eip-2982).
```

Reply:
```suggestion
To understand the motivation of introducing the Proof-of-Stake consensus see the Motivation section of [EIP-2982](/EIPS/eip-2982#motivation).
```

Reply:
@poojaranjan why remove the anchors?  I find them helpful.